src/Files.App/Views/SplashScreenPage.xaml.cs

```diff
@@ -11,17 +11,25 @@ namespace Files.App.Views
 	/// </summary>
 	public sealed partial class SplashScreenPage : Page
 	{
+		private string BranchLabel =>
+			ApplicationService.AppEnvironment switch
+			{
+				AppEnvironment.Dev => "Dev",
+				AppEnvironment.Preview => "Preview",
+				_ => string.Empty,
+			};
+
 		public SplashScreenPage()
 		{
 			InitializeComponent();
 		}
 
-		private void Image_ImageOpened(object sender, RoutedEventArgs e)
+		private void SplashScreenImage_ImageOpened(object sender, RoutedEventArgs e)
```

Comment:
Do we really need these events? I think @d2dyno1 suggested we remove them.

Reply:
I can remove. I don't know what is this TCS doing actually.

Reply:
I removed it in my PR

Reply:
Ok then, I’d like this to be merged.

Reply:
If you could rename it back to what it was, to avoid any future conflicts, if it's not needed 🙂

Reply:
Sure.